src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -12,22 +12,33 @@
 (defn create-board []
   (into [] (repeat 9 "_")))
 
+(defn create-4x4-board []
+  (into [] (repeat 16 "_")))
+
 (def numbered-board
   ["0" "1" "2" "3" "4" "5" "6" "7" "8"])
 
+(def numbered-4x4-board
+  ["0" "1" "2" "3" "4" "5" "6" "7" "8" "9" "10" "11" "12" "13" "14" "15"])
+
 (defn format-board [board]
   (apply str (flatten (for [part (partition 3 board)] (flatten (map vector part '(" | " " | " "\n---------\n")))))))
 
+(defn format-4x4-board [board]
+  (apply str (flatten (for [part (partition 4 board)] (flatten (map vector part '(" | " " | " " | " "\n--------------\n")))))))
+
```

Comment:
@devlinglasman Ah yeah sorry, I should have mentioned that somewhere. I was advised to start off by keeping them separate hence the lack of DRY code! I also have an nxn ticket, so I'll do the clean-up then, but at the moment keeping both types of board separated allows me to see what I'll need to modify when I come to that point.

Reply:
If you can spot duplication, know how to remove it and have the time to do it I would go for it. 👍 

Doing things _later_ is always risky

Reply:
@fabientownsend Yes but because I can't check that it works I figured it'd be better to still have easy access to a working version for now (I guess I could delete it and hunt it down on Github if needed...) - But because I have the nxn ticket, I will have to remove it later anyway!